cns/cnsclient/apiclient.go

```diff
@@ -4,5 +4,7 @@ import "github.com/Azure/azure-container-networking/cns"
 
 // APIClient interface to update cns state
```

Comment:
This API should be called before we start the APIServer Reconcile loop. 

Also Requestcontroller should pass the ContainerNetworkRequest built from getting a crd (if any, else it will be mull) and map of OrchastratorContext (PodName+Podnamespace)->PodIPs (only the pods which have allocated ips). 

Requestcontroller should not be aware of internal IPAM config state.